src/fsharp/symbols/Exprs.fs

```diff
@@ -530,11 +586,67 @@ module FSharpExprConvert =
                 let argR = ConvExpr cenv env arg
                 E.ILFieldSet(None, typR, fspec.Name, argR) 
 
+            | TOp.ILAsm([ ], _), _, [arg] -> 
+                ConvExprPrim cenv env arg
+
+            | TOp.ILAsm([ I_box _ ], _), [ty], [arg] -> 
+                let op = mkCallBox cenv.g m ty arg
+                ConvExprPrim cenv env op
+
+            | TOp.ILAsm([ I_unbox_any _ ], _), [ty], [arg] -> 
+                let op = mkCallUnbox cenv.g m ty arg
+                ConvExprPrim cenv env op
 
-            | TOp.ILAsm([ AI_ceq ], _), _, [arg1;arg2]  -> 
+            | TOp.ILAsm([ I_isinst _ ], _), [ty], [arg] -> 
+                let op = mkCallTypeTest cenv.g m ty arg
+                ConvExprPrim cenv env op
+
+            | TOp.ILAsm([ I_ldtoken _ ], _), [ty], _ -> 
+                let op = mkCallTypeOf cenv.g m ty
+                ConvExprPrim cenv env op
+
+            | TOp.ILAsm([ EI_ilzero _ ], _), [ty], _ -> 
+                E.DefaultValue (ConvType cenv ty)
+
+            | TOp.ILAsm([ AI_ldnull; AI_cgt_un ], _), _, [arg] -> 
+                let elemTy = tyOfExpr cenv.g arg
+                // let op = mkCallIsNotNull cenv.g m elemTy arg
```

Comment:
Remove the comment please

Reply:
@forki Can we keep it until the final review? In case another reviewer might like the IsNotNull op instead of NotEquals null?

Reply:
Removed.